ftw/upgrade/utils.py

```diff
@@ -193,7 +195,27 @@ def log_memory_usage(logger):
         'Current memory usage in MB (RSS): {:0.1f}'.format(rss))
 
 
-def optimize_memory_usage():
+LOAD_LIMITS = {'virtual_memory_available': 100 * 1024 * 1024,
```

Comment:
Could you drop the `virtual_` prefix from these please? I remember this already bugged me in the nightly jobs, but for some reason I didn't press the issue. But it's really quite a misnomer, because virtual memory is the entirety of physical memory + swap.

Whereas with `available` and `rss` the whole point is that swap is excluded from the calculation.

`psutil` lists all those stats under `virtual_memory()` because technically they all are a part of virtual memory management, but a name like `virtual_memory_available` lead me to first think "Oh-oh, are we measuring / comparing the wrong thing here?"